docs/contributing/README.md

```diff
@@ -30,6 +30,11 @@ The full text of our Code of Conduct is available in our repository: <https://gi
 ## Development Environments
 
 We use a Docker-based container management system called [Lando](https://lando.dev/) to manage reusable development environments to ensure everyone is working with the same dependencies. To get started spinning up a development environment, you'll roughly need to follow these steps:
+
+::: warning Windows
+If you're using Windows as a development environment, please refer to the documentation on [setting up WSL before proceeding](./wsl.md). WSL is highly recommended for Windows development.
```

Comment:
Looks good, but I noticed on the WSL page: 
"Install nvm and use that to install node.js and npm in WSL.
nvm install page: [https://github.com/nvm-sh/nvm]
Use nvm to istall node rather than the distribution's included Node.js package"

But I don't remember needing to install node. Is this still needed?

Reply:
@wreality may be able to speak better to this but I'll investigate. 

Good spot on that typo too, by the way.

Reply:
This may be outdated with Ubuntu lastest.  With previous versions lando required a newer version of node than came packaged in the distro.  I can only guess that the version in the distro is newer now and therefore works without installing node separately.

(TLDR, feel free to delete the step about installing nvm and node.)

Reply:
Ok, thank you for the quick response. I'll delete that step.